packages/web-react/src/components/Toast/README.md

```diff
@@ -353,9 +353,11 @@ const { show } = useToast();
          │                 ┌─⫸ ToastBar ID (required)
          │                 │
 show('Toast message', 'toast-id', {
+  autoCloseInterval: 3000  // Set interval after ToastBar will be closed, default: 3000
   color: 'danger',         // Color variant, default: 'inverted'
-  iconName: 'download',    // Name of a custom icon to be shown along the message, default: undefined
+  enableAutoClose: true    // If true, ToastBar will close after `autoCloseInterval`, default, true
```

Comment:
You can use the magic word `Whether` -> `Whether to close ToastBar after the autoCloseInterval runs out`.

But in this case, I see that you are trying to be consistent with the rest. Just think that the use of the `whether` sounds better.